src/Gateway.php

```diff
@@ -269,8 +302,23 @@ public function init_form_fields() {
 				),
 				'default'     => '',
 			],
-			'icon'                => [
+			'icon_display'        => [
 				'title'       => __( 'Icon', 'pronamic_ideal' ),
+				'type'        => 'select',
+				'options'     => [
+					'default' => \__( 'Default', 'pronamic_ideal' ),
+					'none'    => \__( 'None', 'pronamic_ideal' ),
+					'custom'  => \__( 'Custom', 'pronamic_ideal' ),
+				],
+				'description' => sprintf(
+					'%s%s',
+					$description_prefix,
+					__( 'This controls the icon which the user sees during checkout.', 'pronamic_ideal' )
```

Comment:
Same description as `icon` setting.

Reply:
Improved in 7878b1033e7c60cabca3d6715bc00625dab4c731.